shell/server/proxy.js

```diff
@@ -1620,8 +1614,96 @@ Proxy.prototype.handleRequest = function (request, data, response, retryCount) {
       }, context);
     } else if (request.method === "DELETE") {
       return session.delete(path, context);
+    } else if (request.method === "PROPFIND") {
+      return session.propfind(path, {
+        mimeType: request.headers["content-type"] || "application/octet-stream",
```

Comment:
I'm curious why it's important to set a default Content-Type, and wondering if we can lose this logic.